### PR TITLE
Docs: readme updates + logo

### PR DESCRIPTION
**Pull request by IceKhan13**
### Summary

Docs: readme updates + logo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,33 +7,69 @@ PurpleCaffeine: tracking of quantum programs and experiments
...
+    trial.add_circuit("circuit", circuit)
+    trial.add_operator("obs", obs)
+
+    # run
+    exp_value = Estimator().run(circuit, obs, shots=shots).result().values.item()
+    
+    # track results of run
+    trial.add_metric("exp_value", exp_value)
+
+local_backend.get("Example trial")
```
Comment (mickahell) [inline README.md]: Now it's with an id, for local backend it's a mix between date and name.

File: client/README.md [hunk 2]
```diff
@@ -1,3 +1,99 @@
...
+    trial.add_circuit("circuit", circuit)
+    trial.add_operator("obs", obs)
+
+    # run
+    exp_value = Estimator().run(circuit, obs, shots=shots).result().values.item()
+    
+    # track results of run
+    trial.add_metric("exp_value", exp_value)
+
+local_backend.get("Example trial")
```
Comment (mickahell) [inline client/README.md]: Same as the other one

File: docs/images/readme_logo.png
Comment (mickahell) [inline docs/images/readme_logo.png:1]: 🚀

**Review by mickahell — CHANGES_REQUESTED** [review]
Good to me :) just a thing about the get, now it's with an id (mix of name and date) :

https://github.com/IceKhan13/purplecaffeine/blob/900729a7a2f3f4145c28724b4f9814d2faaf9e15/client/purplecaffeine/core.py#L346

**Comment by IceKhan13** [comment]
Removing get for now, because it is not intuitive. We will think about it :)

**Review by mickahell — APPROVED** [review]
Todo bien para mi :)

**Pull request merged**